server/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -358,24 +358,17 @@ public long getMinFileGeneration() {
 
 
     /**
-     * Returns the number of operations in the translog files that aren't committed to lucene.
+     * Returns the translog generation of the last index commit.
      */
-    public int uncommittedOperations() {
-        return totalOperations(deletionPolicy.getTranslogGenerationOfLastCommit());
-    }
-
-    /**
-     * Returns the size in bytes of the translog files that aren't committed to lucene.
-     */
-    public long uncommittedSizeInBytes() {
-        return sizeInBytesByMinGen(deletionPolicy.getTranslogGenerationOfLastCommit());
+    public long uncommittedGeneration() {
```

Comment:
can this be removed (preferably) or made package private for testing? I don't want production code to use this - it just adds circular dependencies.

Reply:
I've removed this from `Translog` and moved this to `TestTranslog`.

Reply:
Finally, I prefer using translog stats :). All removed.